src/index.js

```diff
@@ -1,45 +1,116 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Kindly remove this comment that prevents the linter to apply to your file, it will help you get benefit of `eslint` 😄

Reply:
This has been fixed, thank you.